docs/csharp/whats-new/snippets/whats-new-csharp9/FitAndFinish.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NewCsharp9
+{
+    public class WeatherForecastOptions
+    {
+
+    }
+
+    public class WeatherForecast
+    {
+
+    }
+
+    public class WeatherStation
+    {
+        public string Location { get; init; }
+
+        // <WeatherStationField>
+        private List<WeatherObservation> observations = new();
+        // </WeatherStationField>
+
+        // <ForecastSignature>
+        public WeatherForecast ForecastFor(DateTime forecastDate, WeatherForecastOptions options)
+        // </ForecastSignature>
+        {
+            // implementation elided
+            return new();
+        }
+    }
+
+    public class FitAndFinish
+    {
+        public static void Examples()
+        {
+            // <InitWeatherStation>
+            WeatherStation station = new() { Location = "Seattle, WA" };
```

Comment:
I had an ah-ha moment when looking at this, and I am VERY curious - this looks a lot like object initializers syntax, but omitting the type. With that said, in object initializers you could omit the `()` on the object, for example:

Code like this:

```csharp
WeatherStation station = new WeatherStation() { Location = "Seattle, WA" };
```

Could omit the `()`, like this:

```csharp
WeatherStation station = new WeatherStation { Location = "Seattle, WA" };
```

I'm curious, can we do the same thing now with the `new` syntax?

```csharp
WeatherStation station = new { Location = "Seattle, WA" };
```

I'm wondering because this looks like an anonymous object, instead of the `WeatherStation`... but wow, that would be really cool!

Reply:
```csharp
using System;

Console.WriteLine();

Student x = new { FirstName = "Youssef" };

public class Student
{
    public string FirstName { get; set; }
}
```

>    error CS0029: Cannot implicitly convert type '<anonymous type: string FirstName>' to 'Student'

Looks like the parentheses cannot be omitted.